easybuild/easyconfigs/q/QUAST/QUAST-4.4-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'QUAST'
+version = '4.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://quast.sourceforge.net/quast'
+description = """QUAST evaluates genome assemblies.
+
+QUAST works both with and without a reference genome. 
+The tool accepts multiple assemblies, thus is suitable for comparison."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/ablab/%(namelower)s/archive/']
+sources = ['%(namelower)s_%(version)s.tar.gz']
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('Boost', '1.61.0'),
+    ('matplotlib', '1.5.1', '-Python-%(pyver)s')
+]
+
+options = {'modulename': 'site'}
```

Comment:
@ysagon Better clarify this with a comment, it's a bit of a hack since `import site` will always work (since `site` is a part of the Python standard library).

We should have a cleaner way of bypassing the `import` check...

Reply:
Just in case someone picks up on this: you can easily skip the default `import` check using `False` here:

```python
options = {'modulename': False}